pkg/esp32_sdk/patches/0020-freertos-portasm-changes-for-RIOT-for-riscv32.patch

```diff
@@ -0,0 +1,65 @@
+From a32436c8bc11a0b956d95fbdd535e83e004347d5 Mon Sep 17 00:00:00 2001
+From: Gunar Schorcht <gunar@schorcht.net>
+Date: Thu, 10 Mar 2022 13:10:23 +0100
+Subject: [PATCH 20/20] freertos/portasm: changes for RIOT for riscv32
```

Comment:
What are those changes / why are they needed?
Would be valuable information when updating the SDK in the future

Reply:
I suppose you mean comments in the commits as I provided in the fix.

Reply:
May I squash?

Reply:
Sure, squash away